tests/conftest.py

```diff
@@ -225,9 +225,11 @@ def test_settings(test_dir, mocker, live_mock_server):
     wandb_dir = os.path.join(test_dir, "wandb")
     mkdir_exists_ok(wandb_dir)
     # root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+    time_stamp: float = time.time()
+    datetime_now: datetime.datetime = datetime.datetime.fromtimestamp(time_stamp)
     settings = wandb.Settings(
-        _start_datetime=datetime.datetime.now(),
-        _start_time=time.time(),
+        _start_datetime=datetime_now,
+        _start_time=time_stamp,
         api_key=DUMMY_API_KEY,
```

Comment:
curious: why we can't use `settings._set_time_stamps()` here?

Reply:
sure, good call.